### ISSUE TITLE
opening images without UI overlay

### ISSUE THREAD

**Issue by ichier** [tuskyapp/Tusky#3336]
when opening an image it shows full-screen but is overlapped by UI. you need to tap once, to hide UI. that's rally bad. i want to see the full image instantly, not just parts of it - i saw parts in the timeline before! (it's worse now, that ALT-text was made more readable.)
only if i am curious what the alt-text would be, i would like to tap in order to show the UI. or if' i want to save or share the image.
default should be showing the image without any UI to avoid hundreds of thousands of unnecessary taps.
for people using screen-readers an option could be implemented to not show images at all - but instantly the alt text. (if those are using tusky at all...)

note that i'm using a tablet horizontally and opening an image shows it way bigger, so i do open potentially nice pictures quite often.

also: tapping on the "void" next to the image closes the image again. this leads to even more stupid taps. especially on vertical images. imho only swiping up and down (and tapping a back button) should close the image, while tapping anywhere on screen should turn UI on and off; that way everything would be more consistent and intuitive.

* * * *
- Tusky Version: 21

- [x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Lakoja** [comment]
This happens (is problematic) for every image?

I was entering the same issue a few days ago as bug because it didn't occur to me to try tapping...

I didn't really send the issue once I noticed the tapping and that it only is a problem for very high images (on a smartphone).

**Comment by ichier** [comment]
no, not every one, but - in my case: tablet horizontal - every vertical one.

on the phone it depends on your screen-size and image-ratio.
but even if it's not escaping viewport, showing images that big, stays a matter of taste. i'd prefer to have an overview. so an option like "show images in in timeline in full if possible" would be a nice option for this new behavior, even if you'd shrink/crop images, that are too big to match viewport.

also: if a post has like 4 images it's becoming ugly quite easily, no matter of device-orientation.
in that case you could optionally provide a carousel for instant full images-experience - would be a cool feature for some.
but i'd still prefer tuskys previous behavior, its most compact but in  good way, providing all information at a glance.

(fun-fact: "enlarging" a vertical image then actually shrinks it, in order to show it in full)
